db/migrate/20181207202634_add_cat_genome.rb

```diff
@@ -0,0 +1,25 @@
+class AddCatGenome < ActiveRecord::Migration[5.1]
```

Comment:
Do you know what tables we use migrations to add new data to? 
No problem in starting without proper background?